tests/e2e-pw/specs/merchant/merchant-admin-account-balance.spec.ts

```diff
@@ -0,0 +1,128 @@
+/**
+ * External dependencies
+ */
+import { test, expect } from '@playwright/test';
+
+/**
+ * Internal dependencies
+ */
+import { useMerchant } from '../../utils/helpers';
+
+test.describe(
+	'Merchant account balance overview for single deposit currency accounts',
+	() => {
+		// Use the merchant user for this test suite.
+		useMerchant();
+
+		test(
+			'View the total and available account balance for a single deposit currency (USD)',
+			{
+				tag: '@critical',
+			},
+			async ( { page } ) => {
+				await test.step(
+					'Navigate to the Payments Overview screen',
+					async () => {
+						await page.goto(
+							'/wp-admin/admin.php?page=wc-admin&path=/payments/overview'
+						);
+					}
+				);
+
+				await test.step(
+					'Observe the total account balance, ensuring it has a dollar value with the correct currency formatting',
+					async () => {
+						const totalBalanceValue = page.getByLabel(
+							'Total balance',
+							{
+								exact: true,
+							}
+						);
+
+						// Match the total balance value to the USD format $1*
+						// Intentionally not expecting a specific value to allow for different values in different environments.
+						await expect( totalBalanceValue ).toHaveText( /\$\d+/ );
+					}
+				);
+
+				await test.step(
+					'Observe the available account balance, ensuring it has a dollar value with the correct currency formatting',
+					async () => {
+						const availableFundsValue = page.getByLabel(
+							'Available funds',
+							{
+								exact: true,
+							}
+						);
+						await expect( availableFundsValue ).toHaveText(
+							/\$\d+/
+						);
+					}
+				);
+			}
+		);
+	}
+);
+
+test.describe(
+	'Merchant account balance overview for multiple deposit currency accounts',
+	{
+		tag: '@critical @todo',
+	},
+	() => {
+		/**
+		 * Test requirements not yet met:
+		 * - A merchant account with multiple deposit currencies must be available in our e2e environment
+		 */
```

Comment:
I don't have a strong opinion on this case in particular, but this comment could also be defined through a [test annotation](https://playwright.dev/docs/test-annotations#annotate-tests). With test annotations, we can add descriptions, notes, links, etc to tests or describe blocks, and these will appear in the test report.

It would be great if we could define a rule to determine when we should rely on annotations for adding additional details to our tests or simple use comments in our code for it. The difference basically is that regular code comments are not shown in the test report and test annotations are.

Maybe we could use comments to describe the test implementation itself and annotations for everything else? (dependencies, testing strategy, attaching images, linking technical debt...).

Reply:
To show this comment in the test report, I've moved this description of the `@todo` blocker in an annotation in 938b8f8, alongside an `issue` annotation.